content/applications/marketing/email_marketing/analyze_metrics.rst

```diff
@@ -17,83 +20,148 @@ locations.
+
+  A high bounce rate could indicate issues either with the mailing list used for the mailing or with
+  the sender authentication.
+
+  A low bounce rate may indicate that the mailing list used is up-to-date and the sender
+  authentication is trusted by email providers. View the
+  :ref:`email-marketing/deliverability-issues` section for more information.
+
 .. tip::
    Click on the smart buttons to see the recipient records for each metric.
 
-.. _email-marketing/key-metrics:
+.. _email_marketing/create_reports:
 
-Opened rate
------------
+Create metrics reports
+======================
 
-The *Opened* rate measures the percentage of emails opened by recipients against the total number of
-sent emails.
+Individual metrics can be analyzed by creating a report. To begin, click on the smart button of the
+desired metric.
 
-A high open rate may indicate that the subject line was timely, compelling, and successfully
-prompted the recipients to view the message.
+Next, click the :icon:`fa-caret-down` :guilabel:`(down arrow)` to the right of the search bar to see
+the list of filtering and grouping parameters.
 
-A low open rate may indicate the subject line failed to capture the recipients' interest. It could
-also indicate the email ended up in a spam or junk folder due to having a poor sender reputation or
-failing to :doc:`configure the proper DNS records <../../general/email_communication/email_domain>`.
+:guilabel:`Filters`, located in the left column of the search options, can be used to keep only the
+results that fit the filter. For example, selecting the :guilabel:`Bounced` filter only shows emails
+that could not be delivered.
 
-Replied rate
-------------
+:guilabel:`Group By`, found in the middle column, is used to organize the results into groups, and
+can be used with or without filters.
 
-The *Replied* rate measures the percentage of recipients who responded to the email directly against
-the total number of sent emails.
+.. note::
+  Setting multiple :guilabel:`Group By` options creates nested groups, according to which option
+  is selected first. For example, selecting :guilabel:`Mailing Lists`, followed by
+  :guilabel:`Bounce`, in the :guilabel:`Group By` column, sorts all results *first* by the mailing
+  list, *then* by whether an email was bounced.
 
-A high replied rate may indicate the email resonated with recipients, prompting them to take action
-or provide feedback.
+  This can be verified by looking at the direction, and order, of the selections in the group tile
+  that appears in the search bar.
 
-A low replied rate may suggest that the message lacked relevance or did not contain a clear call to
-action.
+.. example::
+   A monthly newsletter has been sent out, and 6.9% of the sent emails were bounced.
 
-Clicked rate
-------------
+   .. image:: analyze_metrics/newsletter-metrics.png
+      :align: center
+      :alt: The metrics smart buttons of the newsletter.
```

Comment:
Please correct me if I'm mistaken, but shouldn't the screenshot being used here show 6.9% in the 'Bounced' smart button?